__tests__/helpers/Signatures.re

```diff
@@ -0,0 +1,183 @@
+module Mappable = {
```

Comment:
`Signatures.rei` is redundant with this.

Reply:
Ok. Removing.

Reply:
See https://github.com/glennsl/rebase/pull/4#discussion_r594535520